scripts/receipts.sh

```diff
@@ -61,10 +62,11 @@ function main() {
 
   tools::install
 
-  receipts::generate "${build}" "${buildReceipt}"
-  receipts::generate "${run}" "${runReceipt}"
+  # We are generating receipts for all platforms but copying the amd64 receipts as $buildReceipt and $runReceipt
+  receipts::generate::multi::arch "${build}" "${run}"
```

Comment:
Does the code still work if the images are not multi-arch?

Reply:
Yes. I created the stack (oci images) using a branch without multi-arch support and just confirmed that the receipts are generated correctly.

Reply:
Verification:
You can see that the multi-arch images are approximately twice the size.
```
root@896b3302e73a:/workspace# pwd
/workspace
root@896b3302e73a:/workspace# ls
LICENSE  README.md  build  buildpack_integration_test.go  go.mod  go.sum  init_test.go  integration  metadata_test.go  scripts  stack
root@896b3302e73a:/workspace# ls -hl build
total 487M
-rw-r--r-- 1 root root 160M Aug 10 19:05 build.oci
-rw-r--r-- 1 root root 301M Jul 28 18:39 build.oci.multi-arch
-rw-r--r-- 1 root root 9.6M Aug 10 19:05 run.oci
-rw-r--r-- 1 root root  18M Jul 28 18:39 run.oci.multi-arch
root@896b3302e73a:/workspace# 
```

Here is the output from running the script on the amd64 (only) oci archives.
```
root@896b3302e73a:/workspace# scripts/receipts.sh 
Using jam 2.4.0
  Uploading build image to 127.0.0.1:64316/build
  Uploading run image to 127.0.0.1:64316/run

Generating package SBOM for build.oci
Generating CycloneDX package SBOM using syft for build.oci on platform linux/amd64
 ✔ Parsed image                                                                        sha256:09d3ee6d18a1c2fe2044cac159d456d7f36da6f1c7108a97dc1b095cab1e83bb
 ✔ Cataloged packages              [182 packages]  

Generating package SBOM for run.oci
Generating CycloneDX package SBOM using syft for run.oci on platform linux/amd64
 ✔ Parsed image                                                                        sha256:caeae21d310d9c5a6dd8f19b01e8e34e5ce656835572699fa24a0ae91a7f6367
 ✔ Cataloged packages              [8 packages]  
Success! Receipts are:
  /workspace/build/build-amd64-receipt.cyclonedx.json
  /workspace/build/build-receipt.cyclonedx.json
  /workspace/build/run-amd64-receipt.cyclonedx.json
  /workspace/build/run-receipt.cyclonedx.json

root@896b3302e73a:/workspace# 
```

Reply:
Related - can we delete the `receipts::generate()` function? It doesn't seem to be used anywhere now that we have the new function `receipts::generate::multi::arch()`.